src/clients/HubPoolClient.ts

```diff
@@ -805,6 +805,14 @@ export class HubPoolClient extends BaseAbstractClient {
     return { runningBalance, incentiveBalance };
   }
 
+  public isValidChainId(_chainId: number, mainnetBlockToCheck: number): boolean {
```

Comment:
ooc, why is this `_chainId` ? Normally the underscore is to hint the linter that the variable is intentionally unused, but it is used here.

Reply:
Just so it isn't confused with `this.chainId`.